benchmark/main.cpp

```diff
@@ -203,20 +222,24 @@ int main(int argc, char* argv[]) {
         useInt64 = (find(result.begin(), result.end(), "64") != result.end());
     }
 
-    cout << "Column size in KB,Data type,Time in ns,Cache,Thread Count" << endl;
+    cout << "Column size in KB,Data type,Time in ns,Cache,Thread Count,DB type" << endl;
     for (auto size: DB_SIZES){
         cerr << "benchmarking " << (size / 1024.0f) << " KiB" << endl;
+
+        // For smallest size, iterations will be 1 / factor, e.g. 1/0.01 = 100. For double the size half of that etc.
+        int mIterations = iterations == 0 ? max(1, (int) (1 / ITERATIONS_FACTOR / size * DB_SIZES[0])) : iterations;
```

Comment:
I still find this super confusing. Why are we using the reciprocal value and not a multiplicative factor? Alternatively, we could define a minimum amount of data to be benchmarked against.
Also, what does the `m` in `mIterations` stand for?

Reply:
I agree that a reciprocal value is not really intuitive.
But the greater problem I see is that with the current setting we will run the benchmark only once for DB_SIZES > 1 MB. Therefore I would propose (6 iterations like before as a minimum):
`iterations = iterations == 0 ? max(6, (int) (ITERATIONS_FACTOR / size * DB_SIZES[0])) : iterations;` with ITERATIONS_FACTOR = 100.0f

Reply:
If you don't like to overwrite iterations just rename it, maybe dynamic or modified Iterations

Reply:
Always 6 at least? Ok sure, why not. I thought reciprocal value would be more intuitive, but I can change it.

Reply:
The reason I'm not overwriting iterations is that otherwise `iterations == 0` will fail in successive loops. Will rename it.